### ISSUE TITLE
License

### ISSUE THREAD

**Issue by iblancasa** [iputils/iputils#93]
Where is the license?

Thank you :+1:

**Comment by okias** [comment]
In the c files

**Comment by iblancasa** [comment]
Ok, thanks.

I think would be a good idea to have a LICENSE file. It would be less confuse

**Comment by okias** [comment]
Yes, I agree, feel free to send pull request, just beware, every utility from iputils suite has different license

**Comment by okias** [comment]
Pushed license informations, hope it's OK now :)
